Pipfile

```diff
@@ -8,6 +8,6 @@ Pyphen = "*"
 "repoze.lru" = "*"
 
 [dev-packages]
-pycodestyle = "*"
-pytest = "*"
```

Comment:
hi @cclauss, you appear to have dropped `pytest` as a dependency here. Travis will still pass as it's preinstalled there, but we will need to keep this for local tests.

Reply:
Sorry!!!